modules/ROOT/partials/glossary.adoc

```diff
@@ -1,117 +1,132 @@
-[[cmd]]$cmd:: Cypher administrative command, for example, `DROP DATABASE`.
-[[component]]$component:: Component name.
-[[constr]]$constr:: Constraint name.
-[[constrDescrOrName]]$constrDescrOrName:: Constraint descriptor or name.
-[[context]]$context:: Freeform description of a context.
-[[coordinates]]$coordinates:: Coordinate value.
-[[count]]$count:: Amount
+[[clause]]$clause:: A Cypher clause, for example, `USE`, `MATCH`.
+[[cmd]]$cmd:: A Cypher command, for example, `DROP DATABASE`, `PROFILE`, `SHOW PROCEDURES`.
+[[component]]$component:: Date-time component, for example, `year`, `month`, `hour`, etc.
+// we need to change the parameter $component in 51N27 to `$feat` maybe.
+[[constr]]$constr:: Constraint name, for example, `book_isbn`.
+//Change $constr to $constrName
+[[constrDescrOrName]]$constrDescrOrName:: Constraint description, `(:Label {property0, property1, property2, property1, property3})`.
+//Change $constrDescrOrName to $constrDescr
+[[context]]$context:: Freeform description of a context, where something is invalid or unsupported, `OPTIONS`, `URL`, `username`, etc.
+[[coordinates]]$coordinates:: Coordinate value, for example, `[3.15, 4.012, 5.905]`.
+[[count]]$count:: Number of something, for example, `3`.
 [[countAllocs]]$countAllocs:: Desired number of servers to use.
 [[countSeeders]]$countSeeders:: Number of seeding servers.
-[[crs]]$crs:: Coordinate reference system, for example, WGS84.
+//To check with the clustering team about $countAllocs and $countSeeders.
+[[crs]]$crs:: Coordinate reference system, for example, `WGS84`.
 [[db]]$db:: Database name, for example, `myDb`.
-[[dim]]$dim:: Number representing index dimenionality.
-[[edition]]$edition:: Freeform edition description.
-[[endpointType]]$endpointType:: One of `start`, `end`.
+[[dim]]$dim:: Number representing index or point dimenionality.
+[[edition]]$edition:: Neo4j edition, for example, Community or Enterprise.
+[[endpointType]]$endpointType:: One of `start` or `end` of a relationship.
 [[entityId]]$entityId:: ID of a node or a relationship.
 [[entityType]]$entityType:: A node or a relationship.
 [[expr]]$expr:: Cypher expression, for example, `1 + n.prop`.
-[[exprType]]$exprType:: Freeform expression type.
+[[exprType]]$exprType:: Freeform expression type, for example, `EXISTS`, `COUNT`, `COLLECT`.
 [[feat]]$feat:: Freeform feature description.
-[[field]]$field:: Input field, for example, for a procedure or deserialization.
-[[format]]$format:: Duration format.
-[[fun]]$fun:: Function name.
-[[funClass]]$funClass:: User defined function implementation class name.
+[[field]]$field:: Input field, for example, of a procedure, function, or deserialization.
+//52N22 and 52N36 to change $field to $arg. Check 22N98 with Antonio.
+[[fun]]$fun:: Function name, for example, `collect()`.
+[[funClass]]$funClass:: User-defined function implementation class name, for example, `MyFunctionClass`.
 [[graph]]$graph:: The name of a graph, for example, `myGraph`.
```

Comment:
Out of curiosity, is this parameter for GDS-related features?
How do we (in the team-errors) define and distinguish _graph_ and _database_? 
We have these terms in [the Neo4j Glossary](https://neo4j.com/docs/neo4j-glossary/#_g). Are those definitions still true and valid?

Reply:
I believe `graph` here is unrelated to the `graph` in GDS, both referencing the scientific concept that Neo4j builds on https://en.wikipedia.org/wiki/Graph_theory.

@nadja-muller Do you have a more detailed explanation for the `graph` vs `database` question? I attempted an example with composite but that felt more confusing than helpful 😅

Reply:
It's a bit tricky. I think what we ended up on was:
* graph: logical object, data in the database container e.g. once you access the data, you try to access the graph (which is why it is used as a target of the USE clause)
* database: container/file of the graph (which is why it is used in the administration commands)
I'm not sure how to write this out formally, however. There are very close to each other.

Reply:
https://development.neo4j.dev/docs/neo4j-glossary/#_g

Reply:
@l-heemann, are you happy with this PR? May I merge it?